ui/src/app/job-details/resources/resources.component.html

```diff
@@ -20,5 +20,10 @@
         {{ logFileData.get(logFile) }}
       </div>
     </div>
+    <div *ngIf="currentTabId == 'event-detail'">
+      <div *ngFor="let eventDetail of eventDetails" class="inline-text">
+          {{ eventDetail.startTime + ' ' + eventDetail.description }}
```

Comment:
Does this put each event on a separate line?

Reply:
Yes, should I use a javascript native loop or use `*ngFor`  on a DOM? Or concatenate the string in ts file?

Reply:
I like the `*ngFor`